docker-compose.ci.yml

```diff
@@ -1,6 +1,28 @@
 version: '3'
 
 services:
+  test_browser:
+    build:
+      dockerfile: ./Dockerfile
+      args:
+        CI_COMMIT_SHA: ${IMAGE_TAG}
+    image: elifesciences/elife-xpub:${IMAGE_TAG}
+    # no need to start the up in any scenario, yet
```

Comment:
the `app`? but it is started now, outdated comment?

Reply:
@giorgiosironi not sure what you mean...

Reply:
done